src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -262,7 +262,7 @@ private bool ValidatePath(TrackedDownload trackedDownload)
             if ((OsInfo.IsWindows && !downloadItemOutputPath.IsWindowsPath) ||
                 (OsInfo.IsNotWindows && !downloadItemOutputPath.IsUnixPath))
             {
-                trackedDownload.Warn("[{0}] is not a valid local path. You may need a Remote Path Mapping.", downloadItemOutputPath);
+                trackedDownload.Warn("[{0}] is not a valid local path. You may need a Remote Path Mapping. See the remote path entry on the download troubleshooting article for details.", downloadItemOutputPath);
```

Comment:
Similar to my first comment, article while accurate doesn't seem like the best word to use, since they probably don't even know where to find it.